helpers/config/config_struct.go

```diff
@@ -686,6 +686,10 @@ func validateCredHubSettings(config *config) error {
 }
 
 func validateVolumeServices(config *config) error {
+	if config.IncludeVolumeServices == nil {
+		return nil
+	}
+
 	if !config.GetIncludeVolumeServices() {
```

Comment:
I don't think that we need a `nil` check for both `config.IncludeVolumeServices` and `config.GetIncludeVolumeServices()`, as they should be pulling the same value.

Reply:
I _think_ the issue is that `config.IncludeVolumeServices` is a pointer to a bool, so negating `config.GetIncludeVolumeServices()` will fail if the underlying pointer is `nil`.

The expression could probably be collapsed into something like:
`if config.GetIncludeVolumeServices() != nil && !config.GetIncludeVolumeServices() {`

Reply:
Ok, if that's the case, then it's fine as is.